package.json

```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0"
+    "vue-router": "^4.0.0-0",
+    "vueperslides": "^3.2.0"
```

Comment:
This might conflict with Sanaa's change (as she is using the same library). Naman and I will talk about this with Sanaa.